### PR TITLE
Add org.avidemux.Avidemux

### PR DESCRIPTION
**Pull request by eszlari**

### CODE REVIEW

**Comment by eszlari** [comment]
Could drop nasm if https://github.com/flatpak/freedesktop-sdk-base/issues/21 is fixed.

File: org.avidemux.Avidemux.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+app-id: org.avidemux.Avidemux
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.10 # fails to build with 5.11
```
Comment (barthalion) [inline org.avidemux.Avidemux.yaml]: 5.11 mostly cleaned up some includes, see Arch patch: https://git.archlinux.org/svntogit/packages.git/tree/trunk/avidemux-qt5.11.patch?h=packages/avidemux

**Comment by barthalion** [comment]
bot, build org.avidemux.Avidemux

**Comment by eszlari** [comment]
```
(flatpak-builder:23769): flatpak-builder-ERROR **: Failed to load included manifest (/srv/buildbot/worker/build-x86_64-3/org.avidemux.Avidemux/shared-modules/glu/glu-9.0.0.json): Failed to open file '/srv/buildbot/worker/build-x86_64-3/org.avidemux.Avidemux/shared-modules/glu/glu-9.0.0.json': No such file or directory
```

**Comment by nedrichards** [comment]
bot, build org.avidemux.Avidemux

**Comment by AsciiWolf** [comment]
Still the same issue.

**Comment by AsciiWolf** [comment]
bot, build org.avidemux.Avidemux

**Comment by TingPing** [comment]
bot, build org.avidemux.Avidemux

**Comment by AsciiWolf** [comment]
bot, build org.avidemux.Avidemux

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.avidemux.Avidemux

**Comment by nedrichards** [comment]
@eszlari anyone else we can add?

**Pull request closed without merging**